docs/deployment/FPC_on_Azure.md

```diff
@@ -0,0 +1,139 @@
+# How to use an Azure Confidential Computing Instance to run FPC in HW mode
+Date of writing 14.03.2022.
+This guide is based on the following sources.
+[Koshi Ikegawa](https://qiita-com.translate.goog/ikegawa-koshi/items/8cf1fef1004fc16beb15?_x_tr_sl=ja&_x_tr_tl=en&_x_tr_hl=de&_x_tr_pto=wapp#fpc%E3%81%AE%E3%83%93%E3%83%AB%E3%83%89)
+
+- [How to use an Azure Confidential Computing Instance to run FPC in HW mode](#how-to-use-an-azure-confidential-computing-instance-to-run-fpc-in-hw-mode)
+  - [Creating the Confidential Computing Instance on Azure.](#creating-the-confidential-computing-instance-on-azure)
+  - [Prerequisites](#prerequisites)
+  - [Registering for the SGX Attestation Service Utilizing EPID](#registering-for-the-sgx-attestation-service-utilizing-epid)
+    - [Known Issues](#known-issues)
+  - [Building the FPC dev container](#building-the-fpc-dev-container)
+  - [Final preparation of the dev-container](#final-preparation-of-the-dev-container)
+
+## Creating the Confidential Computing Instance on Azure.
+Use the [Quick Create Portal](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/quick-create-portal) to create
+the virtual machine.
```

Comment:
For a better git experience with Markdown, I would prefer writing a full sentence in a single line. The current FPC main README is a bad example, my apologies. :)

Reply:
Sure, I did not really think about the git expirience tbh. But that is something I should be doing for markdown latex etc as well. Good point ;)